apps/starter/src/ensemble/collections.yaml

```diff
@@ -17,3 +17,19 @@ View:
                     navigateModalScreen:
                       name: Ask Embrace
                       maskClosable: false
+        - Row:
+            mainAxis: spaceBetween
+            crossAxis: center
+            children:
+              - Markdown:
+                  text: "### Groups"
+              - MultiSelect:
```

Comment:
Just sanity checking here, there's no existing widget in Ensemble that has this multi select functionality right?

Reply:
yes, right